docs/self-managed/operational-guides/troubleshooting.md

```diff
@@ -103,12 +103,13 @@ AWS_REQUEST_CHECKSUM_CALCULATION=WHEN_REQUIRED
 
 This disables the additional checksum calculation in the S3 client and should resolve the issue.
 
-## Zeebe Backup with Azure Blob Storage
+## Zeebe backup with Azure Blob storage
```

Comment:
I'd argue that Azure Blob Storage should stay capitalized: https://azure.microsoft.com/en-us/products/storage/blobs

Reply:
Yes -- good catch. I will adjust across all instances in the docs.